examples/feature_demo/plot_pbr.py

```diff
@@ -40,7 +44,11 @@
 # Load meshes, and apply env map
 # Note that this lits the helmet already
 gltf_path = (
-    Path(__file__).parent / "models" / "DamagedHelmet" / "glTF" / "DamagedHelmet.gltf"
+    Path(__file__).parents[1]
```

Comment:
The [sphinx-gallery docs](https://sphinx-gallery.github.io/stable/faq.html#why-is-file-not-defined-what-can-i-use) say that one could use `os.getcwd()` instead of `__file__`. That would mean the examples *must* be run from the containing folder. So we could do:
```py
try:
    this_dir = os.path.dirname(__file__)
except NameError:
    this_dir = os.getcwd()
```

Reply:
That does the trick @almarklein. I've expanded this a bit to make it less magical when a user encounters it in the gallery, and - for now - I will add the following block to each example that uses `__file__`:

![image](https://user-images.githubusercontent.com/4402489/204131974-5d017b31-4cd3-4814-851b-cc08eff6bd39.png)

With this, the examples now render successfully :)

![image](https://user-images.githubusercontent.com/4402489/204131996-2aabdc6f-dcdf-4bbd-aeb0-3d8567170c8d.png)